tl.tl

```diff
@@ -6998,6 +7008,7 @@ local function add_compat_entries(program: Node, used_set: {string: boolean}, ge
    local function load_code(name: string, text: string)
       local code: Node = compat_code_cache[name]
       if not code then
+         -- TODO: does this need a parse_lang?
```

Comment:
I don't think this _needs_ it because we only load well-known and well-behaved code here (the compatibility chunks listed below in `add_compat_entries`), but since these are Lua chunks, it could be a good idea to explicitly pass `"lua"` as the `parse_lang`, if only for documentation sake.